catalog/ui/catalog-ui-search/src/main/webapp/component/property/property.collection.view.js

```diff
@@ -210,7 +210,6 @@ define([
             propertyIntersection = propertyIntersection.filter(function(property) {
                 return (!properties.isHidden(property)
                 && property.indexOf('metacard') === -1
```

Comment:
I'm thinking we should remove the validation and metacard filters also.  These can be hidden attributes instead.  Some integrators may want these attributes displayed.